README.md

```diff
@@ -1,2 +1,8 @@
 # NeuralNetworks-FinalProject
 Final project for EN.605.647.8VL.FA22
+
+## Team Spaces
+
+*Microsoft Teams*: https://teams.microsoft.com/l/team/19%3a6JMAAUaGHQSo39X7t2o5axoERQDkb4GbrkDfEDnimDQ1%40thread.tacv2/conversations?groupId=41f9507c-e507-411b-8ea9-66e268dfe503&tenantId=9fa4f438-b1e6-473b-803f-86f8aedf0dec
```

Comment:
Do we want this to be part of a public repo?

Reply:
Ah I missed that, thought this was private repo.